Documentation/concepts/kubernetes/configuration.rst

```diff
@@ -47,6 +47,18 @@ to your preferences:
   packets that involve an L4 connection state change. Valid options are
   ``none``, ``low``, ``medium``, ``maximum``.
 
+  - ``none`` - Generate a tracing event on every receive and send packet.
+  - ``low`` - Generate a tracing event on every send packet.
+  - ``medium`` - Generate a tracing event on every new connection and connection
+    state change, and periodically generate tracing events on new sets of TCP
+    flags seen for existing TCP connections. The reporting interval for new TCP
+    flags is defined by ``monitor-aggregation-interval``.
```

Comment:
```suggestion
  - ``medium`` - Generate at least one event for each new connection, at least once per ``monitor-aggregation-interval``, and also generate one event for each packet that contains TCP flags that were not previously seen. Each direction tracks TCP flags and report interval separately. 
```

Reply:
Ehh probably still needs more work.

Events are only ever triggered based on packets occurring, so if there are no packets then there is no event during a `monitor-aggregation-interval`.

At the same time, the "new flags" logic ignores the `monitor-aggregation-level`. So we can't say that "`monitor-aggregation-level` ensures that a maximum of one event is sent during the interval", because the flags logic could cause multiple events to be sent within the interval.

We could say that it amortizes to one event per interval, assuming that packets are sent during the interval...?

Reply:
`i++`

```suggestion
  - ``medium`` - Generate a tracing event on every new connection, any time a packet contains TCP flags that have not been previously seen for the packet direction, and on average once per ``monitor-aggregation-interval`` (assuming that a packet is sent during the interval). Each direction tracks TCP flags and report interval separately. 
```